kinbot/mess.py

```diff
@@ -537,7 +602,13 @@ def write_barrier(self, reaction, index, left_zeroenergy, right_zeroenergy, barr
         if left_zeroenergy < 0 or right_zeroenergy < 0:
             tun = ''
         else:
-            tun = self.tunneltpl.format(cutoff=min(left_zeroenergy, right_zeroenergy),
+            if self.par['pes']:
+                tun = self.tunneltpl.format(cutoff='{cutoff}',
+                                        imfreq='{imfreq}',
+                                        welldepth1='{left_zero}',
+                                        welldepth2='{right_zero}')
+            else:
+                tun = self.tunneltpl.format(cutoff=min(left_zeroenergy, right_zeroenergy),
```

Comment:
tunneling left as keywords to fill in for PES, so that for each uq iteration new energy levels can be calculated and input as needed.